lib/seattleflu/id3c/cli/command/etl/redcap_det_swab_n_send.py

```diff
@@ -0,0 +1,575 @@
+"""
+Process REDCap DET documents into the relational warehouse.
+
+Contains some hard-coded logic for which project ID correlates to which project
+(e.g. 17421 is the PID for Shelters)
+"""
+import os
+import re
+import click
+import json
+import hashlib
+import requests
+from typing import Any, List, Dict, Optional, Match, Collection
+from datetime import datetime, timezone
+from id3c.db.session import DatabaseSession
+from id3c.db.datatypes import Json
+from id3c.cli.command.etl import etl, race
+
+from id3c.cli.command.etl.redcap_det import (
+    COMPLETE_AND_VERIFIED,
+    LOG,
```

Comment:
I don't think we want to share the `LOG`. We will probably want to differentiate the log output between the different redcap-det etl processes so we can debug things more easily if something goes wrong.

Reply:
Oh, right. I had originally written that temporary import thinking the contents of this file would go in the same file as `redcap-det`. Of course this is no longer the case, so I have to change that.